.github/workflows/ci.yml

```diff
@@ -179,6 +179,28 @@ jobs:
           name: macOS (${{ matrix.python }})
           flags: macOS,${{ matrix.python }}
 
+  # run CI on a musl linux
+  Alpine:
+    name: "Alpine"
+    runs-on: ubuntu-latest
+    container: alpine
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install necessary packages
+        run: apk update && apk add python3-dev bash nodejs musl-dev
```

Comment:
maybe good to add a comment saying why these packages in particular are needed, or if it was suggested by someplace online a link to that.